src/Compilers/VisualBasic/Portable/Binding/Binder_Conversions.vb

```diff
@@ -1579,7 +1579,7 @@ DoneWithDiagnostics:
 
             ' We have a successful tuple conversion rather than producing a separate conversion node 
             ' which is a conversion on top of a tuple literal, tuple conversion is an element-wise conversion of arguments.
-            Dim isNullableTupleConversion = (convKind = ConversionKind.WideningNullable) Or (convKind = ConversionKind.NarrowingNullable)
+            Dim isNullableTupleConversion = (convKind And ConversionKind.Nullable) <> 0
```

Comment:
It may be more robust to have extension methods like "IsNullable".
Possibly applies to InterpolatedString as well

Could be done as a separate fix/issue, but it seems a good thing to do.